src/components/inspector/panels/components/StatPanel.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { memo } from 'react'
+import ColorsControl from '~components/inspector/controls/ColorsControl'
+
+const StatPanel = () => {
+  return (
+    <>
+      <ColorsControl label="Color Scheme" name="colorScheme" />
```

Comment:
I think you can remove this control because is not implement in Chakra UI side : 
![image](https://user-images.githubusercontent.com/1102595/190378582-605bdbdd-b546-4046-8499-89764e17a3cc.png)